src/test/java/redis/clients/jedis/PipeliningTest.java

```diff
@@ -342,7 +344,8 @@ public void waitAof() {
 
     try (Jedis j = new Jedis(endpoint.getHostAndPort())) {
       j.auth(endpoint.getPassword());
-      assertEquals("aof", j.get("wait"));
+      Awaitility.await().atMost(5, TimeUnit.SECONDS).pollInterval(50, TimeUnit.MILLISECONDS)
+          .untilAsserted(() -> assertEquals("aof", j.get("wait")));
```

Comment:
:heart:

Reply:
Love it. Got to add it to Lettuce tests too at some point!